anomaly_detector/adapters/som_model_adapter.py

```diff
@@ -113,7 +113,8 @@ def infer(self, false_positives=None):
             now = datetime.datetime.now()
             # Get data for inference
             data, json_logs = self.storage_adapter.load_data(config_type="infer", false_positives=false_positives)
-            if data is None:
+            if (data is None) or len(data) == 0:
+                logging.info("No data returned. Sleeping for 5 seconds")
```

Comment:
When fetching from elasticsearch it will return EmptyDataFrame. Which will not get caught by `(data is None)` -> False.
We should check the length is a more legit check.